adhocracy-plus/assets/js/captcheck.js

```diff
@@ -0,0 +1,121 @@
+window.onload = function () {
```

Comment:
Is this file mostly copied? Then we should add the license text

Reply:
Yes, I just changed it in a way that the api url is not hard coded and adapted it to our linting rules. Will add the license